### ISSUE TITLE
required algorithm refers to a named curve, not signing algorithm

### ISSUE THREAD

**Issue by OR13** [digitalbazaar/ecdsa-sd-2023-cryptosuite#7]
https://github.com/digitalbazaar/ecdsa-sd-2023-cryptosuite/blob/main/lib/requiredAlgorithm.js#L4

I think you mean ES256... P-256 is a curve, not an algorithm.

**Comment by OR13** [comment]
See algorithms at the bottom of the readme...

https://github.com/transmute-industries/vc-di-sd

**Comment by dlongley** [comment]
Yeah, doesn't work for us with EdDSA not indicating key strength. Will look into what we can do without adding complexity.
